k8s/operators/src/diskpool/main.rs

```diff
@@ -672,14 +716,14 @@ fn error_policy(error: &Error, _ctx: Arc<OperatorContext>) -> Action {
         when.to_rfc2822(),
         duration.as_secs()
     );
+    info!("requeue after :{:?}", duration);
```

Comment:
we already have a message above, why is this needed?

Reply:
Will remove it . It was logger i added for testing